Objects/descrobject.c

```diff
@@ -1962,9 +1966,8 @@ property_get___isabstractmethod__(propertyobject *prop, void *closure)
 }
 
 static PyGetSetDef property_getsetlist[] = {
-    {"__name__", (getter)property_get__name__, (setter)property_set__name__},
-    {"__isabstractmethod__",
-     (getter)property_get___isabstractmethod__, NULL,
+    {"__name__", property_get__name__, property_set__name__, NULL, NULL},
```

Comment:
This makes me wonder: what's your reason for adding the `NULL`s?
(C only needs one value and zeroes the rest, and many of these structs have more rarely used stuff at the end -- though perhaps the reasons for that are more historical than ergonomic.)

Reply:
I feel that if we have a half-way type, it's somewhat harder for new readers to know that there's still the NULL doc field and closure. I think I'm unfortunately not constant in where I'm adding those NULLs so feel free to remove them (but since I was anyway modifying the line, I took the initiative of keeping them).

Reply:
Adding `.doc=NULL` might be more helpful -- it would point out *what* is missing.
The closure is very rarely needed; IMO it's best to leave it out.

Anyway, it's an unimportant style nitpick; please consider it for the future edits :)